carbonmark-data/components/pages/RetirementTrendsPage/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Trans } from "@lingui/macro";
+/**
+ * A UI layout component to position Retirement Trends pages content
+ */
+export default function RetirementTrendsPage(props: {
+  children: React.ReactNode;
+}) {
+  return (
+    <div>
+      <h1>
+        <Trans>Retirement trends</Trans>
+      </h1>
+      <a href="/trends/by-pool">
```

Comment:
Should these be intl links instead?

Reply:
> Should these be intl links instead?

Yeah, I was getting lazy. These are just placeholders before we develop the design for those links.